objcgen/embedder.cs

```diff
@@ -175,7 +177,7 @@ public int Generate (List<string> args)
 					if (File.Exists (filename))
 						return universe.LoadFile (filename);
 				}
-				throw ErrorHelper.CreateError (13, $"Can't find the assembly '{resolve_args.Name}', referenced by '{resolve_args.RequestingAssembly.FullName}'.");
+				throw ErrorHelper.CreateError (13, $"Can't find the assembly '{resolve_args.Name}', referenced by '{resolve_args.RequestingAssembly.FullName}' with directories {String.Join (" ", directories.ToArray ())}.");
```

Comment:
This line makes a test fail. Take a look at the bots. We can update the test to check the message in a diff way.